### PR TITLE
Add handshaking

### PR DESCRIPTION
**Pull request by optimumtact**
## About The Pull Request
You can now shake hands

## Why It's Good For The Game
https://www.oranges.net.nz/~oranges/webm/business_executed.webm

:cl: oranges
add: Added hand shakes
del: Removed tail pull
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
So true

**Comment by Mothblocks** [comment]
:-1: this is going to make it more annoying to hold someones hand and hug them :-1:

**Comment by Colovorat** [comment]
Your PR is missing proper format, an inappropriate title, or contains incorrect/missing information in your PR body.

Please look over your PR and make sure all items are filled out in a way that someone looking back years from now can easily access and understand.

**Comment by RaveRadbury** [comment]
What about the *handshake emote?

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -417,16 +417,12 @@
 		if(HAS_TRAIT(src, TRAIT_BADTOUCH))
 			to_chat(M, span_warning("[src] looks visibly upset as you pat [p_them()] on the head."))
 
-	else if ((M.zone_selected == BODY_ZONE_PRECISE_GROIN) && !isnull(src.getorgan(/obj/item/organ/tail)))
-		SEND_SIGNAL(src, COMSIG_CARBON_TAILPULL, M)
```
Comment (Kylerace) [inline code/modules/mob/living/carbon/carbon_defense.dm:421]: you didnt remove this signal

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -417,16 +417,12 @@
...
-		M.visible_message(span_notice("[M] pulls on [src]'s tail!"), \
-					null, span_hear("You hear a soft patter."), DEFAULT_MESSAGE_RANGE, list(M, src))
-		to_chat(M, span_notice("You pull on [src]'s tail!"))
-		to_chat(src, span_notice("[M] pulls on your tail!"))
-		if(HAS_TRAIT(src, TRAIT_BADTOUCH)) //How dare they!
-			to_chat(M, span_warning("[src] makes a grumbling noise as you pull on [p_their()] tail."))
-		else
-			SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "tailpulled", /datum/mood_event/tailpulled)
```
Comment (Kylerace) [inline code/modules/mob/living/carbon/carbon_defense.dm:429]: or this mood event

**Review by Kylerace — CHANGES_REQUESTED** [review]

**Comment by Misdoubtful** [comment]
I like the part where you can hear other people shaking hands. Like its just that aggressive.

**Comment by optimumtact** [comment]
> I like the part where you can hear other people shaking hands. Like its just that aggressive.

Canonically sounds like this https://www.oranges.net.nz/~oranges/webm/business_executed.webm

**Comment by optimumtact** [comment]
> What about the *handshake emote?

what about it?

**Comment by Mothblocks** [comment]
my WORKFLOW this should be *handshake then offer like the other stuff especially since we already have a family handshake

**Comment by optimumtact** [comment]
no

**Comment by san7890** [comment]
this is like cloning removel 2

**Comment by CRITAWAKETS** [comment]
Title is incorrect.

Should be Add**s** handshaking.

File: code/datums/quirks/negative.dm [hunk 3]
```diff
@@ -833,10 +833,10 @@
 	hardcore_value = 1
 
 /datum/quirk/bad_touch/add()
-	RegisterSignal(quirk_holder, list(COMSIG_LIVING_GET_PULLED, COMSIG_CARBON_HUGGED, COMSIG_CARBON_HEADPAT, COMSIG_CARBON_TAILPULL), .proc/uncomfortable_touch)
+	RegisterSignal(quirk_holder, list(COMSIG_LIVING_GET_PULLED, COMSIG_CARBON_HUGGED, COMSIG_CARBON_HEADPAT), .proc/uncomfortable_touch)
 
 /datum/quirk/bad_touch/remove()
-	UnregisterSignal(quirk_holder, list(COMSIG_LIVING_GET_PULLED, COMSIG_CARBON_HUGGED, COMSIG_CARBON_HEADPAT, COMSIG_CARBON_TAILPULL))
+	UnregisterSignal(quirk_holder, list(COMSIG_LIVING_GET_PULLED, COMSIG_CARBON_HUGGED, COMSIG_CARBON_HEADPAT))
```
Comment (necromanceranne) [inline code/datums/quirks/negative.dm:839]: Add the bad touch check grr

Comment (optimumtact) [reply code/datums/quirks/negative.dm:839]: get real

**Comment by LemonInTheDark** [comment]
> my WORKFLOW

SO TRUE BESTIE

**Comment by Mothblocks** [comment]
oranges sounded like he woul dmake this an offer but if he doesnt by the time this is merged ill change it so i can hold hands and hug

**Comment by EricZilla** [comment]
When you handshake someone they should get a shitty "Ecomony" or "Yes, business" moodlet thats +1 for like 15 seconds

**Comment by Zytolg** [comment]
Someone needs to block this until Oranges gets his tail pulled. _We all know what he is after all_

**Comment by Coastfront** [comment]
We are a pullrequest closer to making it so you can handshake off the captains arm as a hulk in a particularly agressive deal sealing.

**Comment by carshalash** [comment]
Leave it forever to upset the redditors. You know you want to.

**Comment by PowerfulBacon** [comment]
merge it

**Comment by Iamgoofball** [comment]
shaking hands while wearing a suit and tie should give you a bonus moodlet for BUSINESS

**Comment by NanoCats** [comment]
How do you trigger the handshake? I also think it's funny that this removes THE THING as a funny coincidence.

**Comment by UnokiAs** [comment]
The title and about pull request should contain "Replacing tail pulling by handshake" or something

**Comment by Mickyan** [comment]
This would work great if it could be done by using the offer item key with an empty hand

**Comment by itseasytosee** [comment]
Thank you for this wonderful PR oranges :)

**Review by Kylerace — APPROVED** [review]
giphy

**Comment by optimumtact** [comment]
this would break mothblocks workflow

**Comment by LemonInTheDark** [comment]
So true :(

**Comment by itseasytosee** [comment]
I am going to break mothblocks kneecaps.

**Comment by MetalClone** [comment]
Reopen this PR so I can add clown handshake buzzers later thank you.

**Pull request closed without merging**